CoreFarms.cs

```diff
@@ -497,7 +497,8 @@ public void BattleUnderB(string item = "Bone Dust", int quant = 1)
         Core.EquipClass(ClassType.Farm);
 
         Bot.Options.AggroMonsters = true;
-        Core.KillMonster("battleunderb", "Enter", "Spawn", "*", item, quant, false, publicRoom: true, log: false);
+        while (Bot.Inventory.GetQuantity(item) < quant)
```

Comment:
this is redundant Core.CheckInventory already checks for the quantity of bone dust at the start of the method

Reply:
Yes but if the bot reached max stack and it won’t stop. Is there other approach to how to stop it when it reached max stack while farming?

Reply:
Because I want it to farm the exact amount that thrown by the caller, just to save time while farming

Reply:
kill / hunts already have a checkinv for quants withing the function themselves, no needs for an extra check

Reply:
Okay! How do I delete this commit btw 😅

Reply:
Removed this commit